vividus-plugin-websocket/src/main/java/org/vividus/steps/websocket/WebSocketSteps.java

```diff
@@ -53,30 +51,20 @@ public class WebSocketSteps
     private static final Object KEY = WebSocketContainer.class;
     private static final long DEFAULT_TIMEOUT = 60;
 
-    private final Map<String, URI> webSocketConnections;
+    private final Map<String, String> webSocketConnections;
     private final TestContext testContext;
     private final VariableContext variableContext;
     private final ISoftAssert softAssert;
-    private final JettyWebSocketClient webSocketClient;
+    private final StandardWebSocketClient webSocketClient;
```

Comment:
Why was the client changed?

Reply:
That's a long story which starts from https://github.com/vividus-framework/vividus/actions/runs/5950668469/job/16138906235